src/assets/locales/resources_en_US.json

```diff
@@ -1,6 +1,14 @@
 {
-  "skyux_some_key": {
-    "_description": "",
-    "message": ""
+  "skyux_inline_form_cancel": {
```

Comment:
I'd put "button" in the label so it's obvious what this text is being used for.
`skyux_inline_form_button_cancel`, etc.

Reply:
done.